ldk/javascript/examples/javascript-loop/index.js

```diff
@@ -0,0 +1,18 @@
+import { clipboard, whisper } from '@oliveai/ldk'
```

Comment:
These lint errors do not happen locally, but you must first generate a `/dist` directory within the `ldk/javascript` project.

CI IS generating that `/dist` directory in a previous step, but during this lint step this reference cannot be resolved.

We can:
1. Add a lint ignore rule to this file.
2. Ignore the `/examples` dir when running eslint
3. Find how to make sure the `/dist` dir is ready for this lint step on CI (this will take much longer)